Sources/WriteFreely/WFClient.swift

```diff
@@ -328,39 +328,36 @@ public class WFClient {
     ///   - token: The access token for the user moving the post to a collection.
     ///   - postId: The ID of the post to add to the collection.
     ///   - modifyToken: The post's modify token; required if the post doesn't belong to the requesting user.
-    ///   - collectionAlias: The alias of the collection to which the post should be added.
+    ///   - collectionAlias: The alias of the collection to which the post should be added; if `nil`, this removes the post from any collection.
     ///   - completion: A handler for the returned `Bool` on success, or `Error` on failure.
     public func movePost(
         token: String? = nil,
         postId: String,
         with modifyToken: String? = nil,
-        to collectionAlias: String,
+        to collectionAlias: String?,
         completion: @escaping (Result<Bool, Error>) -> Void
     ) {
         if token == nil && user == nil { return }
         guard let tokenToVerify = token ?? user?.token else { return }
 
-        guard let url = URL(string: "collections/\(collectionAlias)/collect", relativeTo: requestURL) else { return }
+        var urlString = ""
+        if let collectionAlias = collectionAlias {
+            urlString = "collections/\(collectionAlias)/collect"
+        } else {
+            urlString = "posts/disperse"
+        }
+        guard let url = URL(string: urlString, relativeTo: requestURL) else { return }
         var request = URLRequest(url: url)
 
         request.httpMethod = "POST"
         request.addValue("application/json; charset=utf-8", forHTTPHeaderField: "Content-Type")
         request.addValue(tokenToVerify, forHTTPHeaderField: "Authorization")
 
-        var bodyObject: [[String: Any]]
+        var bodyObject: [Any]
         if let modifyToken = modifyToken {
-            bodyObject = [
-                [
-                    "id": postId,
-                    "token": modifyToken
-                ]
-            ]
+            bodyObject = collectionAlias == nil ? [ postId ] : [ [ "id": postId, "token": modifyToken ] ]
```

Comment:
Technically the user should never have a `modifyToken` if they're removing a post from a collection (`collectionAlias == nil`)... A post will always be owned by a user in this situation, meaning the `modifyToken` shouldn't be used. If we had made "disperse post" its own func instead of overloading, we'd leave out the `modifyToken` param entirely. So in this case, perhaps we should enforce this logic by returning [an error] if `collectionAlias == nil` here, so clients can know that they're doing something wrong?

Reply:
Ah, good to know. If collectionAlias is nil but modifyToken is not, we can return a completion failure early. New commit incoming!